packages/main/src/TabSeparator.js

```diff
@@ -58,6 +58,14 @@ class TabSeparator extends UI5Element {
 		return true;
 	}
 
+	/**
+	 * Returns the DOM reference of the separator that is placed in the header.
```

Comment:
I suggest to combine into one note:
	 * <b>Note:</b> Tabs and separators, placed in the <code>subTabs</code> slot of other tabs are not shown in the header. Calling this method on such tabs or separators will return <code>undefined</code>.